chpl/chpl-service/src/main/java/gov/healthit/chpl/certifiedproduct/service/ListingService.java

```diff
@@ -123,7 +122,7 @@ public CertifiedProductSearchDetails createCertifiedProductSearchDetailsBasic(Lo
     }
 
     public CertifiedProductSearchDetails createCertifiedProductSearchDetailsBasic(CertifiedProductDetailsDTO dto) throws EntityRetrievalException {
-        CertifiedProductSearchDetails searchDetails = CertifiedProductSearchDetails.builder()
+        CertifiedProductSearchDetails listing = CertifiedProductSearchDetails.builder()
```

Comment:
Not a problem with your code changes in this method, but the method reads to me as though it should return the CertifiedProductSearchBasicDetails object instead of the full details object that it is returning. Thoughts?